friends/src/FriendsList.js

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import axios from 'axios';
+import Friend from './Friend';
+import UpdateFriend from './UpdateFriend';
+import FriendListHeader from './FriendListHeader';
+import { ListWrapper, StyledForm } from './Styled';
+
+class FriendsList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      friends: [],
+      name: '',
+      age: '',
+      email: '',
+      id: ''
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:5000/friends')
+      .then(data => {
+        // console.log(data.data);
+        this.setState({ friends: data.data });
+        // console.log(this.state);
+      })
+      .catch(() => alert('GET Error'));
+  }
+
+  handleInput = event => {
+    event.preventDefault();
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleSubmit = event => {
+    event.preventDefault();
+    // console.log(this.state.friends);
```

Comment:
Another technique you can try out. More info on it [here](https://stackoverflow.com/questions/42307735/destructuring-state-in-react-render)
```suggestion
    const { name, age, email } = this.state

```